File: qualifier-week-3/non graded assignments/PPA6 - Not Graded.c
//Prefix Code(Visible)
#include <stdio.h>
#include <math.h>
int main()
{
    //Initialization of variables
    double principal, annualRate, monthlyRate, monthlyPayment;
    int numMonths;

    //Reading Inputs
    scanf("%lf", &principal);
    scanf("%lf", &annualRate);
    scanf("%d", &numMonths);
    // Write code below to calculate the monthly payment and store the result in monthlyPayment variable
    monthlyRate = (annualRate / 100 ) / 12;

    monthlyPayment = (principal * monthlyRate * pow(1 + monthlyRate, numMonths)) / (pow(1 + monthlyRate, numMonths) - 1);
   //Printing the output
    printf("%.0lf", monthlyPayment);
    return 0;
}

File: qualifier-week-3/non graded assignments/PPA6 - Not Graded.rs
fn main() {
    let mut principal: f64 = 0.0;
    let mut annual_rate: f64 = 0.0;
    let mut num_months: i32 = 0;

    let _ = scanf!("{:f64}", &mut principal);
    let _ = scanf!("{:f64}", &mut annual_rate);
    let _ = scanf!("{:f64}", &mut num_months);

    let monthly_rate = annual_rate / 100;

    let monthly_payment = principal * monthly_rate * (1.0 + monthly_rate).powi(num_months as i32) / ((1.0 + monthly_rate).powi(num_months as i32) - 1.0);

    println!("Monthly Payment: {:.2f}", monthly_payment);
}
